bench/config/templates/Procfile

```diff
@@ -3,7 +3,10 @@ redis_cache: redis-server config/redis_cache.conf
 redis_socketio: redis-server config/redis_socketio.conf
 redis_queue: redis-server config/redis_queue.conf
 {% endif %}
-web: bench serve {% if webserver_port -%} --port {{ webserver_port }} {%- endif %}
+
+# Remove --noreload flag to enable reloading on python file changes.
+# Keep this flag until initial setup on site is finished.
+web: bench serve {% if webserver_port -%} --port {{ webserver_port }} {%- endif %} --noreload
```

Comment:
👍 

Is there a way to start bench in developer mode without touching procfile? can we use `bench start` with option?

Reply:
This PR will likely make `--no-dev` command confusing. So we should fix that too.

Maybe we can have `--dev` command too that enables file reloader. This way we won't have to keep editing the procfile. 

@gavindsouza your thoughts ?